src/components/Image.js

```diff
@@ -133,6 +134,8 @@ export const Image = ({
           <View style={styles.contentContainerStyle}>
             {!!message && <ImageMessage message={message} />}
             {!!button && <ImageButton button={button} />}
+            {!!buttons?.length &&
+              buttons.map((button, index) => <ImageButton key={index} button={button} />)}
```

Comment:
we do not want to use just index for react component keys. can you find a better solution please

Reply:
Yes, I will definitely find a better solution.